packages/jaeger-ui/src/components/App/index.jsx

```diff
@@ -56,19 +56,41 @@ export default class JaegerUIApp extends Component {
           <Router history={history}>
             <Page>
               <Switch>
-                <Route path={searchPath} component={SearchTracePage} />
-                <Route path={traceDiffPath} component={TraceDiff} />
-                <Route path={tracePath} component={TracePage} />
-                <Route path={dependenciesPath} component={DependencyGraph} />
-                <Route path={deepDependenciesPath} component={DeepDependencies} />
-                <Route path={qualityMetricsPath} component={QualityMetrics} />
-                <Route path={monitorATMPath} component={MonitorATMPage} />
+                <Route path={searchPath}>
+                  <SearchTracePage />
+                </Route>
```

Comment:
With this change, the child components, `SearchTracePage` in this example, would no longer receive the Route props.
So, we need to use the `withRouteProps` HOC to get them.